theme.go

```diff
@@ -510,9 +510,11 @@ func ThemeInfo(name string) ThemeDesc {
 // color from the theme.
 // clr - current object color
 // id - color ID in theme
-func RealColor(clr term.Attribute, id string) term.Attribute {
+func RealColor(clr term.Attribute, style string, id string) term.Attribute {
+	ccolor := fmt.Sprintf("%s%s", style, id)
```

Comment:
Just a question: Can you provide an example of `style`? I see they are just concatenated. Is a point between them make a resulting string better: `ccolor := fmt.Sprintf("%s.%s", style, id)`?

Reply:
sure, I can provide some examples. I like the suggestion of changing it to ```<style>.<id>```